packages/plugin-testing/lib/index.ts

```diff
@@ -0,0 +1,106 @@
+
+/* eslint-disable no-unused-vars, class-methods-use-this */
+import {
+  PluginBase, Event as TrackingEvent,
```

Comment:
Why do you alias Event as TrackingEvent here?

Reply:
B/c `Event` is a type for `HTML DOM Event`. When you use `Event` in code it doesn't refer to `itly` event type.

Reply:
Gotcha, I ran into this too but when I explicitly `import { Event } from '@itly/sdk'` it seems to use our Event by default. Does that not work for you?

Reply:
That works